slobrok/src/vespa/slobrok/server/rpchooks.cpp

```diff
@@ -444,49 +455,40 @@ RPCHooks::rpc_lookupManaged(FRT_RPCRequest *req)
     FRT_Values &args = *req->GetParams();
     const char *name = args[0]._string._str;
     LOG(debug, "RPC: lookupManaged(%s)", name);
-    const ManagedRpcServer *found = _rpcsrvmap.lookupManaged(name);
-
-    if (found == nullptr) {
-        req->SetError(FRTE_RPC_METHOD_FAILED, "Not found");
-    } else {
-        FRT_Values &dst = *req->GetReturn();
-        dst.AddString(found->getName().c_str());
-        dst.AddString(found->getSpec().c_str());
+    const auto & visible = _env.globalHistory();
```

Comment:
if you want to use the local monitoring for 'managed' servers, you should probably have a TODO here as well...